network-mux/src/Network/Mux/DeltaQTraceTransformer.hs

```diff
@@ -104,5 +104,18 @@ makePerSizeRecord tt = PSR
 
 -- May want to make this a configuration variable
 
+-- NOTE this interval must be less than the wrap around time of the
+-- `RemoteClockModel`. The remote clock model has a precision of
+-- microseconds.
```

Comment:
```
2^32 / 1e6
4294.967296
```
So 1 hour tops. If you only need 10 seconds then you only need 24 bits and Karl can have 8 bits back! :smile:

Reply:
True - but we may want to do this over 100 or 1000 seconds - to be decided. That would only give Karl 1 bit back, probably not worth it!